tools/build_publishable.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
Let's get in the habit of documenting what a script does at the top of the script.  The name alone doesn't tell me.